src/main/resources/lang/dutch.lang

```diff
@@ -0,0 +1,190 @@
+# Default Dutch language file
+
+# Which version of Skript this language file was written for
+version: @version@
+
+# What null (nothing) should show up as in a string/text
+none: <none>
```

Comment:
```suggestion
none: <niks>
```

Reply:
While I agree that that is the correct translation, all other languages use <none>, changing that for only Dutch doesn't make much sense to me.